sci-biology/bowtie/bowtie-1.1.1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="An ultrafast memory-efficient short read aligner"
+HOMEPAGE="http://bowtie-bio.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bowtie-bio/${P}-src.zip"
+
+LICENSE="Artistic"
+SLOT="1"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+
+IUSE="examples"
+
+DEPEND="app-arch/unzip"
+RDEPEND=""
+
+PATCHES=( "${FILESDIR}"/${P}-buildsystem.patch )
+
+src_prepare() {
+	epatch ${PATCHES[@]}
+}
+
+src_compile() {
+	unset CFLAGS
```

Comment:
I guess a nice way would be to set them empty (`CFLAGS=`) on the `emake` call. And commenting that the Makefile misuses CFLAGS.